src/components/logo/logo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Box,
+  BoxProps,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+interface LogoContainerProps extends BoxProps {
+  logoSrc?: string;
+  logoAlt?: string;
+  width?: string;
+}
+
+const LogoContainer = styled(Box, {
+  name: 'WmeWizardHeader',
```

Comment:
Should this be `WmeLogoContainer` or something similar?

Reply:
yes good catch, copy/paste error